src/utils.py

```diff
@@ -55,3 +56,6 @@ def load_tavily_api_key():
 def setup_tavily_search():
     api_key = load_tavily_api_key()
     os.environ["TAVILY_API_KEY"] = api_key
+
+def load_empty_dataframe(columns=['business_name', 'total', 'date']):
```

Comment:
Minor but I think we can turn this into a property so it looks a bit cleaner.

Reply:
Oh yeah, definitely!